### ISSUE TITLE
Crash when importing pip in script console

### ISSUE THREAD

**Issue by bjax15** [Vector35/binaryninja-api#1032]
Binary Ninja Version: 1.1.1142 Personal, 32782455
Platform: Ubuntu 18.04 LTS

Crash when importing pip in script console.

**Reproduction Steps:**

1. Open Script Console
1. Execute `import pip`

**Console stdout/err:**
I'm getting inconsistent output. I mostly see:
```
malloc(): memory corruption
```
I sometimes see:
```
free(): corrupted unsorted chunks
```

I'm able to import other python packages (os, sys, struct) without crashing.

My host pip version, if it has any relevance:
`pip 9.0.1 from /usr/lib/python2.7/dist-packages (python 2.7)`

**Comment by ChuckFoo** [comment]
I suspect this issue and the one I reported #1042 with plugins causing similar crashes are the same underlying problem.

**Comment by bpotchik** [comment]
Fixed in build 1.1.1229-dev. Proxy support has been added as well.
